spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/cypher/RelationshipDetail.java

```diff
@@ -88,11 +97,16 @@ public boolean isTyped() {
 		return !this.types.isEmpty();
 	}
 
+	public Optional<Properties> getProperties() {
```

Comment:
is this method used somewhere?

Reply:
I guess not or not yet? I'll take it out.